tests/utils/test_asu.py

```diff
@@ -74,7 +75,7 @@ def gemmi_map2asu(H, sg, return_phase_shifts):
     # matches sgtbx
     if return_phase_shifts:
         Hasu_gemmi, ref_isym, ref_phis = gemmi_map2asu(H, sg, return_phase_shifts)
-        phis = np.deg2rad(rs.utils.canonicalize_phases(phic * phis))
+        phis = np.deg2rad(rs.utils.canonicalize_phases(phis))
     else:
         Hasu_gemmi, ref_isym = gemmi_map2asu(H, sg, return_phase_shifts)
```

Comment:
Per https://github.com/project-gemmi/gemmi/issues/215, `Op.negated()` negated the real-space translation of the symmetry operations (not just the rotational component). This had a direct impact on phase shifts, requiring them to be flipped for Friedel symmetry operations, as was done here with `phic * phis`.

By using `op.combine(friedel)`, we can now directly compare the phase shifts.